codegen/cmd_test.go

```diff
@@ -237,9 +237,9 @@ var _ = Describe("Cmd", func() {
 
 		painterNode := node.Content[0].Content[1]
 		enabledMapField := painterNode.Content[0]
-		Expect(enabledMapField.HeadComment).To(Equal("# Enable creation of the deployment/service."))
+		Expect(enabledMapField.HeadComment).To(Equal("# Arbitrary overrides for the component's [deployment\n# template](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/)"))
```

Comment:
Since fields in `values.yaml` are alphabetical ascending ordered, `deploymentOverrides` description is now the first element